Dockerfile

```diff
@@ -11,9 +11,11 @@ RUN apt-get update \
     groff \
     jq \
     less \
+    openssl \
```

Comment:
just curious, when engendered this?

Reply:
ahh it was either istioctl or the helm3 install